packages/mc-scripts/package.json

```diff
@@ -119,6 +122,7 @@
     "@types/moment-locales-webpack-plugin": "^1.2.3",
     "@types/node-fetch": "2.6.11",
     "@types/serve-handler": "^6.1.1",
+    "memfs": "^4.8.2",
```

Comment:
tried using the `mock-fs` for testing first, but it seemed to intervene with the module resolution of webpack. Opted to use this file system mock library, as referenced by the webpack documentation https://webpack.js.org/contribute/writing-a-loader/#testing